test/colortemp_range_missing_allowed.txt

```diff
@@ -27,6 +27,7 @@ LED1923R5
 7602031P7
 8718696167991
 8718696170557
+1742030P7
```

Comment:
Please don't add this but add the color temp range instead as described here: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#31-retrieving-color-temperature-range-only-required-for-lights-which-support-color-temperature

Reply:
Thanks for advice! I've requested the range values via the lamp's topic and added them.

Debug Received Zigbee message from 'Terrasse_Calla_3', type 'readResponse', cluster 'lightingColorCtrl', 
data '{"colorTempPhysicalMax":500,"colorTempPhysicalMin":153}' from endpoint 11 with groupID 0

I'm not sure with regards to the zigbee2mqtt.io documentation. Did I understand it correctly that it will be autogenerated somehow or do I have to add at least the image for examle?

Reply:
Yes it will be auto generated. Feel free to add an image, otherwise I will do it right before the next release (1 June)